test/integration/setup.spec.js

```diff
@@ -1,4 +1,5 @@
-const serve = require('@nebula.js/cli-serve'); // eslint-disable-line
+/* eslint-disable no-undef */
```

Comment:
In a next PR I will try to tackle the integration test setup. Will try to get rid of this as well